ui/app/mainui/SendModalHandler.qml

```diff
@@ -0,0 +1,152 @@
+import QtQuick 2.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
+
+import AppLayouts.Wallet.stores 1.0 as WalletStores
+
+import shared.popups.send 1.0
+import shared.stores.send 1.0
+
+import utils 1.0
+
+QtObject {
+    id: root
+
+    required property var popupParent
+    required property int loginType
+    required property TransactionStore transactionStore
+    required property WalletStores.CollectiblesStore walletCollectiblesStore
+
+    // for ens flows
+    required property string myPublicKey
+    required property string ensRegisteredAddress
+    // TODO: This should probably be a property and not a function. Needs changes on  backend side
+    property var getStatusTokenKey: function() {}
+
+    // for sticker flows
+    required property string stickersMarketAddress
+    required property string stickersNetworkId
+
+    function openSend(params = {}) {
+        let sendModalInst = sendModalComponent.createObject(popupParent, params)
+        if (sendModalInst.opened) {
```

Comment:
This will create a memory leak if we call this again and again and the SendModal is already opened

Reply:
Ah right

Reply:
hmmm thats true, but what about a new request to launch when its already launched? we should handle that, else we can have more than 1 instances open at the same time?

Reply:
What's the chance to open to send modals in the same time? We would have to invoke next one from the currently opened one. None of our popups are guarded against that but it's basically artificial scenario imo.